bindings/python/convert.py

```diff
@@ -97,16 +103,17 @@ def convert_single(model_id: str, folder: str) -> List["CommitOperationAdd"]:
     operations = [CommitOperationAdd(path_in_repo=sf_filename, path_or_fileobj=local)]
     return operations
 
+
 def check_final_model(model_id: str, folder: str):
     config = hf_hub_download(repo_id=model_id, filename="config.json")
     shutil.copy(config, os.path.join(folder, "config.json"))
     config = AutoConfig.from_pretrained(folder)
 
-    _, pt_model = infer_framework_load_model(model_id, config)
-    _, sf_model = infer_framework_load_model(folder, config)
+    _, (pt_model, pt_infos) = infer_framework_load_model(model_id, config, output_loading_info=True)
+    _, (sf_model, sf_infos) = infer_framework_load_model(folder, config, output_loading_info=True)
 
-    pt_model = pt_model
-    sf_model = sf_model
+    if pt_infos != sf_infos:
+        raise ValueError(f"different infos {model_id}")
```

Comment:
Maybe add some info on the differences?

Reply:
It's quite noisy which is why I didn't include it (I mean infos can be quite numerous even on pure PT for transformers).

Doing a proper diff could help mitigate that but also relatively verbose code. I'll give it a stab.

Reply:
Diff added.